### PR TITLE
[docs] Added info about anonymous access to the container registry

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Added info about anonymous access to the container registry at specs and gs.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Added info about anonymous access to the container registry.
impact_level: low
```

### CODE REVIEW

File: deckhouse-controller/crds/doc-ru-module-source.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
                       type: string
                       description: Адрес репозитория образов контейнеров.
                     dockerCfg:
-                      description: Строка с токеном доступа к container registry в Base64.
+                      description: Строка с токеном доступа к container registry в Base64. В случае использования анонимного доступа к container registry — не заполняйте это поле.
```
Comment (mortelumina98) [inline deckhouse-controller/crds/doc-ru-module-source.yaml:23]:
```suggestion
                      description: Строка с токеном доступа к container registry в Base64. В случае использования анонимного доступа к container registry не заполняйте это поле.
```

File: docs/site/_includes/getting_started/global/step_cluster_setup_ru.html [hunk 2]
```diff
@@ -119,10 +119,13 @@
       class="textfield"
       type="text" id="registryDockerCfg"
       name="registryDockerCfg" placeholder=""
-      autocomplete="off" />
+      autocomplete="off" />  
     <span class="info">
       Строка авторизации из файла конфигурации Docker-клиента (в Linux обычно это <code>$HOME/.docker/config.json</code>), закодированная в Base64.<br />Подробнее про этот параметр читайте <a href="/products/kubernetes-platform/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-registrydockercfg">в документации</a>.
     </span>
+    <span class="info">
+      В случае использования анонимного доступа к хранилищу образов контейнеров — не заполняйте это поле.
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/step_cluster_setup_ru.html:127]:
```suggestion
      В случае использования анонимного доступа к хранилищу образов контейнеров не заполняйте это поле.
```

File: docs/site/_includes/getting_started/global/step_cluster_setup_ru.html [hunk 3]
```diff
@@ -119,10 +119,13 @@
       class="textfield"
       type="text" id="registryDockerCfg"
       name="registryDockerCfg" placeholder=""
-      autocomplete="off" />
+      autocomplete="off" />
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/step_cluster_setup_ru.html:122]:
```suggestion
      autocomplete="off" /> 
```

File: docs/site/_includes/getting_started/global/step_cluster_setup_ru.html [hunk 4]
```diff
@@ -119,10 +119,13 @@
...
```
Comment (mortelumina98) [inline docs/site/_includes/getting_started/global/step_cluster_setup_ru.html:122]:
```suggestion
      autocomplete="off" />
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10234](https://github.com/deckhouse/deckhouse/pull/10234) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**